tests/userTests.py

```diff
@@ -137,15 +137,18 @@ def test_status_change_bad_status(self):
     def test_list_users(self):
         """Test getting user list by status."""
         postJson = {"status": "denied"}
-        response = self.app.post_json("/v1/list_users_with_status/", postJson, headers={"x-session-id":self.session_id})
+        response = self.app.post_json("/v1/list_users/", postJson, headers={"x-session-id":self.session_id})
         self.check_response(response, StatusCode.OK)
         users = response.json["users"]
         self.assertEqual(len(users), 1)
 
+        response = self.app.post_json("/v1/list_users/", headers={"x-session-id": self.session_id})
+        self.check_response(response, StatusCode.OK)
```

Comment:
yeah we can add that. since the goal is to check that all users are indeed being returned, i can put the total number of users in the check.